[receiver/aerospike] Flaky test TestAerospikeIntegration

```
2023/01/07 01:30:38 Container is ready id: 7633c5d32c71 image: aerospike/aerospike-server:6.2.0.0
    integration_test.go:84: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/aerospikereceiver/integration_test.go:84
        	            				/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/aerospikereceiver/integration_test.go:296
        	Error:      	Received unexpected error:
        	            	ResultCode: INVALID_NODE_ERROR, Iteration: 0, InDoubt: false, Node: <nil>: Failed to connect to hosts: [[::1]:4[91](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3859771107/jobs/6579653356#step:6:92)59 127.0.0.1:49159]
        	            	ResultCode: INVALID_NODE_ERROR, Iteration: 0, InDoubt: false, Node: <nil>: Node BB9030011AC4202 (127.0.0.1:49159) is not yet fully initialized
        	            	ResultCode: NETWORK_ERROR, Iteration: 0, InDoubt: false, Node: <nil>: network error. Checked the wrapped error for detail
        	            	dial tcp [::1]:49159: connect: connection refused
        	Test:       	TestAerospikeIntegration
--- FAIL: TestAerospikeIntegration (6.[95](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3859771107/jobs/6579653356#step:6:96)s)
FAIL
coverage: 0.0% of statements
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/aerospikereceiver	6.[98](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3859771107/jobs/6579653356#step:6:99)1s
```

See https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3859771107/jobs/6579653356

--- a/receiver/aerospikereceiver/integration_test.go
+++ b/receiver/aerospikereceiver/integration_test.go
@@ -80,8 +80,13 @@ func populateMetrics(t *testing.T, host *as.Host) {
 	clientPolicy.Timeout = 60 * time.Second
 	// minconns is used to populate the client connections metric
 	clientPolicy.MinConnectionsPerNode = 50
-	c, err := as.NewClientWithPolicyAndHost(clientPolicy, host)
-	require.NoError(t, err)
+
+	var c *as.Client
+	var clientErr error
+	require.Eventually(t, func() bool {
+		c, clientErr = as.NewClientWithPolicyAndHost(clientPolicy, host)
+		return clientErr == nil
+	}, 2*time.Minute, 1*time.Second, "failed to populate metrics")
 
 	ns := "test"
 	set := "integration"

